spec/tasks/one_off_spec.rb

```diff
@@ -104,22 +100,69 @@
         let(:no_source_version) { create(:version, project: project, number: "3.0.0", repository_sources: nil) }
         let(:ignored_source_version) { create(:version, project: project, number: "4.0.0", repository_sources: ["Other"]) }
 
-        it "deletes ignored versions but not the project or its other versions" do
-          expect(Project.all).to match_array([project])
-          expect { Rake::Task["one_off:delete_ignored_maven_versions_and_resync_packages"].invoke("yes") }
+        it "deletes ignored versions but not other versions" do
+          expect { Rake::Task["one_off:delete_ignored_maven_versions_and_resync_packages"].invoke("", "yes") }
             .to change(Version, :all).from([maven_source_version, google_source_version, no_source_version, ignored_source_version]).to([maven_source_version, google_source_version])
-            .and not_change(Project, :count)
         end
       end
 
       context "with only ignored versions" do
         let(:no_source_version) { create(:version, project: project, number: "1.0.0", repository_sources: nil) }
         let(:ignored_source_version) { create(:version, project: project, number: "2.0.0", repository_sources: ["Other"]) }
 
-        it "deletes ignored versions and the project" do
-          expect { Rake::Task["one_off:delete_ignored_maven_versions_and_resync_packages"].invoke("yes") }
+        it "deletes ignored versions" do
+          expect { Rake::Task["one_off:delete_ignored_maven_versions_and_resync_packages"].invoke("", "yes") }
             .to change(Version, :all).from([no_source_version, ignored_source_version]).to([])
-            .and change(Project, :all).from([project]).to([])
+        end
+      end
+    end
+  end
+
+  describe "delete_maven_packages_without_versions" do
+    after(:each) do
+      Rake::Task["one_off:delete_maven_packages_without_versions"].reenable
+    end
```

Comment:
Was it complaining about running a task twice in one test?

Reply:
This was pre-emptive; I needed to add a similar `after` to the specs for `delete_ignored_maven_versions_and_resync_packages` because if I ran individual tests they would pass, but if I ran the full block some would fail because it thought the task had already been run. I had assumed similar would be the case for this task since there are multiple tests in the same block.